app/javascript/packs/leafletMap.js

```diff
@@ -39,27 +39,21 @@ export async function setupMap() {
     maxNativeZoom: 19,
   }).addTo(map);
 
-  const [view, buildingPolygons, buildingMarkers] = await Promise.all([
-    getView(),
-    getBuildings(),
-    getBuildingMarkers(),
-  ]);
-
+  const view = await getView();
   setView(view);
-  addPolygons(buildingPolygons);
-  addMarkers(buildingMarkers);
 
   // display indoor information eg. rooms, labels
   await Promise.all([
-    loadGeoJsonFile("/assets/ABC-Building-0.geojson"),
-    loadGeoJsonFile("/assets/ABC-Building-1.geojson"),
-    loadGeoJsonFile("/assets/ABC-Building-2.geojson"),
-    loadGeoJsonFile("/assets/H_0.geojson"),
-    loadGeoJsonFile("/assets/H_1.geojson"),
-    loadGeoJsonFile("/assets/H_2.geojson"),
-    loadGeoJsonFile("/assets/H_3.geojson"),
-    loadGeoJsonFile("/assets/HS_0.geojson"),
-    loadGeoJsonFile("/assets/HG_-1.geojson"),
+    loadGeoJsonFile("/assets/buildings.geojson", setupGeoJsonFeatureOutdoor, getBuildingStyle, true),
```

Comment:
This smells a lot like code duplication.
We should have an array with the filenames somewhere that we can iterate over.